demo/text-field-validators-demos.html

```diff
@@ -28,19 +29,27 @@ <h3>Minlength and Maxlength</h3>
 
 
     <h3>Pattern</h3>
-    <p>The <code>pattern</code> property specifies a regular expression that the element’s value is
+    <p>The <code>pattern</code> property specifies a regular expression that the element’s
+      value is
```

Comment:
Accidental line break?

Reply:
I used "format document" of the VS code that has changed the format. I will use the previous format for the texts.